runtime/src/main/java/org/corfudb/runtime/view/ObjectsView.java

```diff
@@ -48,6 +48,10 @@ public class ObjectsView extends AbstractView {
             new StreamTagInfo(LOG_REPLICATOR_STREAM_NAME,
                 CorfuRuntime.getStreamID(LOG_REPLICATOR_STREAM_NAME));
 
+    public static final String LR_STATUS_STREAM_TAG = "lr_status";
```

Comment:
I moved the definition here to avoid duplicate definitions in the runtime and infra packages.  Keeping multiple copies will be error-prone.  One idea I had in mind was to create a file - LRConstants where we put all constants shared between runtime and infra.

Reply:
This might be a good idea, cause placed here it's kind of out of context, there is no direct relation or usage in this class.